entrypoint.sh

```diff
@@ -89,6 +93,15 @@ ${overlap}"
 		feedback ERROR "The repo dictionary must be sorted"
 		exit 1
 	fi
+
+	if [[ ${CI:-false} == "true" || ${CONTINUOUS_INTEGRATION:-false} == "true" || ${GITHUB_ACTIONS:-false} == "true" ]]; then
+		export IS_PIPELINE="true"
+		# If this is run in a pipeline and INPUT_AUTO_FIX does not equal true, disable autofix. INPUT_AUTO_FIX="true" would force autofix
```

Comment:
This is an assumption on my part that any changes made by an action on a repo get discarded when the run completes. I also assume we can check for a dirty repo in the pipeline. Is this correct?

Reply:
That's true, unless you explicitly add, commit, and push them (which you can do, and is how our autoupdate logic works)

Reply:
Why can't we just rely on the `CI` env var?  In practice I have always seen it properly set in a pipeline

Reply:
That was a ChatGPT-ism. It gave both as options, but if there isn't a significant use case for other than CI, I'm totally game for simplifying it.

Reply:
Removed this and opted for a local check for CI.